tests/unit/transformers/test_categorical.py

```diff
@@ -310,6 +310,24 @@ def test_transform_all_zeros(self):
         with np.testing.assert_raises(ValueError):
             ohet.transform(['b'])
 
+    def test_transform_large(self):
```

Comment:
This seems to be an integration test (black-box testing) rather than a unit test (white-box testing).
I think we should go a bit deeper on the testing and do separated tests for each one of the new methods, and then a specific test on the `transform` to check whether the right transform is being used internally.